### PR TITLE
Create sample-interactive

### PR DESCRIPTION
**Pull request by alexan0308**

### CODE REVIEW

**Comment by pkra** [comment]
Thanks for this! I think this deserves to be in a new folder, maybe "advanced"?  You might want to change the copyright, too :-)

**Comment by alexan0308** [comment]
Ok. Name
**MathJax-examples/MathJax-base/advanced**
     OR
**MathJax-examples/advanced**
?

**Comment by pkra** [comment]
Sorry for the delay. Just `advanced` -- the `MathJax-base` folder should remain for samples coming out of the main project.

**Comment by alexan0308** [comment]
Sorry for the my delay =)). And this branch have view HTML-github.io?

**Comment by pkra** [comment]
There's currently no "live" version of this repo. But I suppose we could turn this into a small jekyll instance. Let's first merge this into the right folder structure and then refactor the repository.

**Comment by alexan0308** [comment]
I downloaded this branch. Examples of it not working. Wanted MathJax.js and fonts.

File: MathJax-base/sample-interactive [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+    extensions: ["tex2jax.js"],
+	messageStyle: "none",
+    jax: ["input/TeX","output/HTML-CSS"],
+    "HTML-CSS": {
+      availableFonts:[],
+      styles: {".MathJax_Preview": {visibility: "hidden"}}
+    }
+  });
+</script>
+<script type="text/javascript" src="../MathJax.js"></script>
```
Comment (pkra) [inline MathJax-base/sample-interactive:20]: This needs to point to `//cdn.mathjax.org/mathjax/latest/MathJax.js`

**Comment by alexan0308** [comment]
ok i use git-client, so I decided to do it all over again)

**Pull request closed without merging**